mysite/settings.py

```diff
@@ -105,7 +107,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = config('TIME_ZONE', default='UTC')
 
 USE_I18N = True
 
```

Comment:
you should not expose your secret key to the public, try to config it and then keep it in .env file

Reply:
I already configured it.